deploy/AzureAppServices/content/install.cmd

```diff
@@ -0,0 +1,21 @@
+@echo off
+
+REM Create home directory for tracer version
+mkdir D:\home\site\wwwroot\datadog\tracer\v1_10_18
+
+REM Copy all dependencies
+xcopy /e D:\home\SiteExtensions\Datadog.Trace.AzureAppServices\Tracer D:\home\site\wwwroot\datadog\tracer\v1_10_18
+
+REM Create logging directory for tracer version
+mkdir D:\home\LogFiles\Datadog\Tracer\v1_10_18
+
+REM Create directory for agent to live
+mkdir D:\home\site\wwwroot\app_data\jobs\continuous\datadog-trace-agent
+
+REM Copy the trace agent
+copy /y trace-agent.exe D:\home\site\wwwroot\app_data\jobs\continuous\datadog-trace-agent
+
+REM enable powershell install script (TODO: Will we need this?)
+powershell.exe -ExecutionPolicy RemoteSigned -File install.ps1
```

Comment:
AAS supports `install.cmd`, `install.ps1`, and a few other extensions. We don't need both if this script does everything we need.

Reply:
Not sure why there's a Powershell install file as well. If memory serves me, there was a time when you had to have a cmd or bat file do the install, and the exec of a Powershell script was just a convenience if you already had a ps script.

Reply:
I was originally testing xcopy and was wondering if some of the tasks we might do would be better served in ps. It was just experimentation.